File: backoffice/src/main/java/io/mateu/erp/client/admin/AdminArea.java
package io.mateu.erp.client.admin;

import io.mateu.ui.core.client.app.AbstractArea;
import io.mateu.ui.core.client.app.AbstractModule;

import java.util.Arrays;
import java.util.List;

/**
 * Created by miguel on 3/1/17.
 */
public class AdminArea extends AbstractArea {

    public AdminArea() {
        super("Admin");
    }

    @Override
    public List<AbstractModule> getModules() {
        return Arrays.asList(new AdminModule());
    }
}

File: backoffice/src/main/java/io/mateu/erp/client/admin/AdminArea.kt
package io.mateu.erp.client.admin

import com.vaadin.icons.VaadinIcons
import io.mateu.mdd.core.app.AbstractArea
import io.mateu.mdd.core.app.AbstractModule
import java.util.*

/**
 * Created by miguel on 3/1/17.
 */
class AdminArea : AbstractArea(VaadinIcons.COGS, "Admin") {

    override fun buildModules(): List<AbstractModule> {
        return Arrays.asList<AbstractModule>(AdminModule())
    }
}
